src/ClientData/include/ClientData/DataManager.h

```diff
@@ -134,6 +139,8 @@ class DataManager final {
   bool collect_memory_info_ = false;
   uint64_t memory_sampling_period_ms_ = 10;
   uint64_t memory_warning_threshold_kb_ = 8ULL * 1024 * 1024;
+
+  std::optional<orbit_client_data::ThreadStateSliceInfo> selected_thread_state_slice_{std::nullopt};
```

Comment:
I would move this line below to the definition of selected_timer_ (20 lines above)